data/build.gradle

```
@@ -29,10 +29,7 @@ dependencies {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
     implementation 'com.squareup.retrofit2:converter-gson:2.3.0'
```

Remove dependency

---

Been there, done that ;)
```suggestion
    implementation 'com.squareup.retrofit2:converter-gson:2.7.0'
```